ERCS/erc-555.md

```diff
@@ -0,0 +1,209 @@
+---
+number: 555
```

Comment:
```suggestion
number: 7571
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename and the Eth Magicians discussion.